test/tf/hcloud/main.tf

```diff
@@ -0,0 +1,46 @@
+provider "hcloud" {
```

Comment:
Picking an arbitrary place but running a `terraform fmt` in the `test/tf/hcloud` directory will line up the equal signs and make this look nicer 😄